bugsnag-android-performance/src/main/kotlin/com/bugsnag/android/performance/AutoInstrumentationCache.kt

```diff
@@ -0,0 +1,23 @@
+package com.bugsnag.android.performance
+
+import android.app.Activity
+import androidx.annotation.RestrictTo
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class AutoInstrumentationCache {
```

Comment:
This needs a constructor to populate `appStartActivities` and `autoInstrument` (from the `Collection`s in `PerformanceConfiguration`).